CompanyAddress.md

```diff
@@ -7,124 +7,73 @@ _______
-| designated_market_areas | An object array of the designated market areas containing the location. Designated market areas may overlap and thus a location may be in more than one designated market area. Designated market areas are specific to the continental US. See below for details on the structure of these objects. **This property may be absent and the array may be empty**. |
-| partial_match | A boolean value indicating that the geocoder did not return an exact match. **This property will always be false for Company Address searches**.|
-
-&nbsp;
-
-Each element of the returned **admin_areas** array will be formatted as follows:
-
-|   |   |
-|---|---|
-| short_name | The abbreviated form of the administrative area's name. This will vary by country and administrative division level. In the US, states (admin_area level 1) will use the state abbreviation (e.g. "GA"), and counties will use the full county name (e.g. "Gwinnett County"). **This property is always present and non-empty.** |
-| long_name | The unabbreviated form of the administrative area's name. **This property is always present and non-empty.** |
-| level | An integer between 1 and 5 inclusive specifying the hierarchal level at which this administrative area resides. In the US, states will be returned with a level of 1 and counties with a level of 2. **This property is always present and non-empty.** |
+## Request Structure
+Requests consist of:
 
-&nbsp;
+| Field Name        | Type   | Required |
+|-------------------|--------|----------|
+|`company_id`      | string | True      |
+|`country` | string | False       |
+|`state` | string | False       |
+|`city` | string | False       |
+|`branch` | string | False       |
+|`postal_code` | string | False       |
 
-Each element of the returned **designated_market_areas** array will be formatted as follows:
+Example cURL request:
 
-|   |   |
-|---|---|
-| name | The name of the DMA, e.g. "Atlanta, GA." **This property is always present and non-empty.** |
-| id | The integer ID of the DMA. E.g. for the Atlanta, GA DMA, the ID is 524. **This property is always present and non-empty.** |
+```
+curl -X POST \
+  https://api.careerbuilder.com/core/geography/companyaddress \
+  -H 'Accept: application/json;version=1.0' \
+  -H 'Authorization: <BEARER_TOKEN>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+          "postal_code": "53211",
+          "company_id": "NC496cd8bb-93bf-4fe1-b9fc-4c966f4f95e5",
+          "city": "Milwaukee",
+          "state": "WI",
+          "branch": "10th Street Comprehensive Treatment Center",
+          "country": "US"
+      }'
+```
 
-&nbsp;
 
-## Sample Response
 
+## Response Structure
+All responses with an HTTP status of 200 will consist of a JSON object with a top-level "data" node. The response consists of company address and other location parameters like country,state,city,postal code,latitude,longitude and branch codes as show in the below sample response.
```

Comment:
need a space between the commas `country,state,city,postal code,latitude,longitude`

Reply:
Also could we not the latitude and longitude are optional on the response?